addons/medical/stringtable.xml

```diff
@@ -3397,12 +3397,24 @@
             <Chinese>AI 死亡前所能承受的傷害程度</Chinese>
         </Key>
         <Key ID="STR_ACE_Medical_MedicalSettings_fatalInjuryCondition_DisplayName">
-            <English>Fatal Injury</English>
+            <English>Fatal Injury Player</English>
         </Key>
         <Key ID="STR_ACE_Medical_MedicalSettings_fatalInjuryCondition_Description">
-            <English>Defines when you can receive a fatal injury</English>
+            <English>Defines when Player can receive a fatal injury</English>
         </Key>
-         <Key ID="STR_ACE_Medical_MedicalSettings_fatalInjuryCondition_InCardiacArrest">
+        <Key ID="STR_ACE_Medical_MedicalSettings_fatalInjuryConditionAI_DisplayName">
+            <English>Instant death AI</English>
+        </Key>
+        <Key ID="STR_ACE_Medical_MedicalSettings_fatalInjuryConditionAI_Description">
+            <English>Defines if AI will be able to die instantly</English>
+        </Key>
+        <Key ID="STR_ACE_Medical_MedicalSettings_unconsciousConditionAI_DisplayName">
+            <English>Forbid unconscious AI</English>
```

Comment:
I feel like this setting is unnecessarily inverted, it should be "Enable AI Unconsciousness"

(However yeah we could merge and fix this in the main PR)

Reply:
Yup, descriptions and such are inverted and funky, I added the conditions in statemachine and those are kinda notes to self to remember what does it really do. The descriptions are logically correct and describe what the condition really does. The values in code should be inverted so the options could be described sensibly.